twilight-util/Cargo.toml

```diff
@@ -26,7 +31,9 @@ builder = ["dep:twilight-model", "dep:twilight-validate"]
 link = ["dep:twilight-model"]
 permission-calculator = ["dep:twilight-model"]
 snowflake = ["dep:twilight-model"]
-full = ["builder", "link", "permission-calculator", "snowflake"]
+signature-validation = ["dep:ed25519-dalek", "dep:hex"]
+signature-validation-extract_interaction = ["signature-validation", "dep:serde_json", "dep:twilight-model"]
```

Comment:
I am not sure of this name, maybe it should just be `extract-interaction` I would like to hear others ideas.

But it should probably be changed to only use `-`.

Reply:
I've replaced the underscore with a dash. I don't mind waiting on other peoples' naming ideas here, I just picked something bad so I could move on with the adding of `cfg`s.